native_client/client.cc

```diff
@@ -150,29 +161,25 @@ GetAudioBuffer(const char* path)
   // Close sox handles
   sox_close(output);
   sox_close(input);
-#endif // __ANDROID__
+#endif // NO_SOX
 
-#ifdef __ANDROID__
+#ifdef NO_SOX
   // FIXME: Hack and support only 16kHz mono 16-bits PCM
   FILE* wave = fopen(path, "r");
 
   size_t rv;
 
   unsigned short audio_format;
   fseek(wave, 20, SEEK_SET); rv = fread(&audio_format, 2, 1, wave);
-  assert(rv == 2);
```

Comment:
Why did you remove all the asserts?

Reply:
They fail on Windows while we get proper values being read. We have other assets at the end, and it's only for specific usecase, I considered it was not worth spending more time than needed

Reply:
OK